### ISSUE TITLE
训练时用yolov5m.pt作为预训练模型，我的标签必须是例子中80个标签中的一种吗，能自定义标签吗？

### ISSUE THREAD

**Issue by zandb2009** [ultralytics/yolov5#8709]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

训练时用yolov5m.pt作为预训练模型，我的标签必须是例子中80个标签中的一种吗，能自定义标签吗？

**Comment by Le0v1n** [comment]
迁移学习(fine-tuning)了解一下

**Comment by UnglvKitDe** [comment]
@zandb2009 No, it is not necessary. Look at the wiki. It explains step by step how to train YOLOv5 with your own data (with any number of classes). You can also find tips for performance improvements in the wiki. Good luck!

**Comment by zandb2009** [comment]
ok，thanks!

**Comment by xiaohangguo** [comment]
不管标签命名，甚至是你的数据，都是可以自定义的，你要调模型的话，还要超参数学习，需要做很多实验

**Comment by glenn-jocher** [comment]
@xiaohangguo thanks for offering your insights! Indeed, you can easily customize your own labels and data when training with YOLOv5. Fine-tuning the model and experimenting with different hyperparameters are essential for achieving the best results. Good luck with your experiments!

**Comment by xiaohangguo** [comment]
> @xiaohangguo thanks for offering your insights! Indeed, you can easily customize your own labels and data when training with YOLOv5. Fine-tuning the model and experimenting with different hyperparameters are essential for achieving the best results. Good luck with your experiments!

Haha, it's been quite a while since I last answered this question. The guy who asked before probably has already trained his own model by now.

**Comment by glenn-jocher** [comment]
@xiaohangguo indeed, time flies! It's always great to see the community engaging and helping each other out. If anyone has further questions or needs assistance, don't hesitate to ask. Happy training! 😊🚀
